test/DynamicEater.test.ts

```
@@ -0,0 +1,134 @@
+			weweBalance = await wewe.balanceOf(otherAccount.address);
+			expect(weweBalance).to.be.eq(0);
+
+			tokenBalance = await token.balanceOf(otherAccount.address);
+			expect(tokenBalance).to.be.eq(0);
+
+			// Get the vested amount
+			const vested = await merge.vestings(otherAccount.address);
+			expect(vested.amount).to.be.eq(141925);
+		});
+
+		it.only("Should calcuate rates with large numbers", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)